File: dataset_reduzido/codenet/Python/Code/s170274782.py
N = int(input())
A = []
for _ in range(2):
    s = list(map(int, input().split()))
    A.append(s)


#0で降りるか、1で降りるか、2で降りるか....どこで降りたとしてもそれまでは全習得

ans = []

count = 0
for i in range(N):
    baselis = A[0]
    groundlis = A[1]
    count = 0
    count = count + sum(baselis[0:i+1]) + sum(groundlis[i:N+1])
    ans.append(count)

print(max(ans))    

File: output/Llama-2-100/codenet/Python/Java/s170274782.java
import java.util.Scanner;

public class s170274782 {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int N = scanner.nextInt();
        List<List<Integer>> A = new ArrayList<>();
        for (int _ = 0; _ < 2; _++) {
            List<Integer> s = new ArrayList<>();
            s.addAll(Arrays.asList(scanner.next().split(" ")));
            A.add(s);
        }

        List<Integer> ans = new ArrayList<>();
        for (int i = 0; i < N; i++) {
            List<Integer> baselis = A.get(0);
            List<Integer> groundlis = A.get(1);
            int count = 0;
            count += sum(baselis.subList(0, i + 1));
            count += sum(groundlis.subList(i, N + 1));
            ans.add(count);
        }

        int max = Collections.max(ans);
        System.out.println(max);
    }

    public static int sum(List<Integer> list) {
        int sum = 0;
        for (int i : list) {
            sum += i;
        }
        return sum;
    }
}
